packages/github_cli/project.bri

```diff
@@ -51,7 +51,7 @@ export function autoUpdate() {
 
     let version = $releaseData
       | get tag_name
-      | str replace '^v' ''
+      | str replace --regex '^v' ''
```

Comment:
@asheliahut I had to adjust the argument here to properly use the regex capabilities of `str replace`. Otherwise it was just doing a raw find and replace.

Reply:
I wonder if there is a way to take this stuff for GitHub and make it generic so we put in the repo name, and get back this part of this and a path for any replacement for things in version.

Reply:
@kylewlacy began to expose possible solutions here: https://github.com/brioche-dev/brioche-packages/discussions/176 to hide this work into a dedicated function that could be static or not (depending of the option chosen).

Reply:
Awesome, Yeah I am not saying this needs to be done here. I am good with accepting. Sorry I have been really focused on work lately.